### PR TITLE
brightness, contrast and label fields

### PR DESCRIPTION
**Pull request by aquatiko**

### CODE REVIEW

File: src/AstroImages.jl [hunk 1]
```diff
@@ -4,7 +4,7 @@ module AstroImages
 
 using FITSIO, FileIO, Images, Interact, Reproject, WCS, MappedArrays
 
-export load, AstroImage, ccd2rgb
+export load, AstroImage, ccd2rgb, set_brightness!, set_contrast!, add_label!, reset!
```
Comment (giordano) [inline src/AstroImages.jl]: Do you expect users to directly call these functions?  If not, why exporting them?

File: src/AstroImages.jl [hunk 2]
```diff
@@ -87,8 +87,14 @@ end
 
 mutable struct Properties{P <: Union{AbstractFloat, FixedPoint}}
     rgb_image::MappedArrays.MultiMappedArray{RGB{P},2,Tuple{Array{P,2},Array{P,2},Array{P,2}},Type{RGB{P}},typeof(ImageCore.extractchannels)}
+    contrast::Float64
+    brightness::Float64
+    label::Array{Tuple{Tuple{Float64,Float64},String},1}
     function Properties{P}(;kvs...) where P
         obj = new{P}()
+        setproperty!(obj, :contrast, 1.0)
```
Comment (giordano) [inline src/AstroImages.jl]: Wouldn't `obj.contrast = 1.0` work?

File: src/AstroImages.jl (hunk 1 above)
Comment (aquatiko) [reply src/AstroImages.jl]: Yes, they will be called directly by the user.

File: src/AstroImages.jl (hunk 2 above)
Comment (aquatiko) [reply src/AstroImages.jl]: I think so, yes.

File: src/AstroImages.jl [hunk 3]
```diff
@@ -87,8 +87,14 @@ end
...
 mutable struct Properties{P <: Union{AbstractFloat, FixedPoint}}
     rgb_image::MappedArrays.MultiMappedArray{RGB{P},2,Tuple{Array{P,2},Array{P,2},Array{P,2}},Type{RGB{P}},typeof(ImageCore.extractchannels)}
+    contrast::Float64
+    brightness::Float64
+    label::Array{Tuple{Tuple{Float64,Float64},String},1}
     function Properties{P}(;kvs...) where P
         obj = new{P}()
+        obj.contrast = 1.0
+        obj.brightness = 0.0
+        obj.label = Array{Tuple{Tuple{Float64,Float64},String}}(undef,0)
```
Comment (giordano) [inline src/AstroImages.jl]: I think `Tuple{Tuple{Float64,Float64},String}[]` would also work, but they should be equivalent, so there is no need to change.

Comment (aquatiko) [reply src/AstroImages.jl]: But the labels can't be changed that way, right? It would we restricting a possible functionality.

**Pull request merged**